### ISSUE TITLE
I run "expo publish", I get this error: Missing module "hermes-engine/win64-bin/hermesc" in the project

### ISSUE THREAD

**Issue by asalha** [facebook/hermes#552]
## Bug Description
I upgraded recently from SDK 41 to SDK 42 and enabled hermes by following the instructions here: https://docs.expo.io/guides/using-hermes/

I am on Windows 11, I run the following EAS build smoothly without any problem:
````
eas build --clear-cache --platform android
````
The apk is generated and it has a reduced size and it is lightening fast at launch.
The problem comes when I make changes to the code and run "expo publish" for OTA updates, it gives the following error:
````
Missing module "hermes-engine/win64-bin/hermesc" in the project.This usually means hermes-engine is not installed. Please verify that dependencies in package.json include "react-native" and run `yarn` or `npm install`.
````
Hermes engine is installed under react-native as seen in the screenshot below:
![hermes-error](https://user-images.githubusercontent.com/4199314/125747625-449a2755-ce34-4d69-8844-a7f2b8cca93a.jpg)

Although not necessary, I installed hermes-engine (npm i hermes-engine) and tried again, same problem.

Hermes version: (0.5.1)
React Native version (if any): 0.63.2
OS version (if any): Windows 11 / Android 30
Platform (most likely one of arm64-v8a, armeabi-v7a, x86, x86_64): x86_64

## Steps To Reproduce
Create a new Expo managed workflow (sdk 42) and enable hermes. Run eas build then "expo publish".

## The Expected Behavior
To make the OTA updates without any error (as hermes-engine is installed under react-native).

**Comment by ecreeth** [comment]
See https://github.com/expo/expo/issues/13626#

**Comment by asalha** [comment]
It seems it has been solved in expo/expo#13626. Thanks
